README.md

```diff
@@ -13,181 +12,243 @@
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
 [![SciML Code Style](https://img.shields.io/static/v1?label=code%20style&message=SciML&color=9558b2&labelColor=389826)](https://github.com/SciML/SciMLStyle)
 
-Catalyst.jl is a symbolic modeling package for analysis and high performance
+Catalyst.jl is a symbolic modeling package for analysis and high-performance
 simulation of chemical reaction networks. Catalyst defines symbolic
 [`ReactionSystem`](https://docs.sciml.ai/Catalyst/stable/catalyst_functionality/programmatic_CRN_construction/)s,
 which can be created programmatically or easily
-specified using Catalyst's domain specific language (DSL). Leveraging
-[ModelingToolkit](https://github.com/SciML/ModelingToolkit.jl) and
+specified using Catalyst's domain-specific language (DSL). Leveraging
+[ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl) and
 [Symbolics.jl](https://github.com/JuliaSymbolics/Symbolics.jl), Catalyst enables
 large-scale simulations through auto-vectorization and parallelism. Symbolic
 `ReactionSystem`s can be used to generate ModelingToolkit-based models, allowing
-the easy simulation and parameter estimation of mass action ODE models, Chemical
+the easy simulation and parameter estimation of mass action ODE models, chemical
```

Comment:
Chemical Langevin is the name though (i.e. Chemical Langevin Equation). Otherwise this should just say "sde" since the normal Langevin equation people talk about is very much not the equation we generate.

Reply:
I think this was a misstake, I was intending to go from "Chemical" to "chemical". Good spot.

Reply:
This does not remove "Chemical" in "Chemical Langevin", it simply changes it to "chemical Langevin"
(I am fairly confident this is the correct spelling? E.g. https://pubs.aip.org/aip/jcp/article/113/1/297/184125/The-chemical-Langevin-equation)